modules/sdk/src/main/scala/org/tessellation/sdk/domain/healthcheck/types/HealthCheckConsensusDecision.scala

```diff
@@ -0,0 +1,7 @@
+package org.tessellation.sdk.domain.healthcheck.types
+
+sealed trait HealthCheckConsensusDecision
+
+final case class PositiveOutcome() extends HealthCheckConsensusDecision
```

Comment:
If there are no constructor arguments, let's use `case object` instead. If some arguments will occur in the future, then we will change it to `case class`.

Reply:
There will be many arguments, to be added in next PRs

Reply:
I know about the `case object` but in that case I'd avoid switching it back and forth just for one PR. I want to model the healthcheck first starting from the most generic/abstract things and then dive deeper and deeper.